src/README.md

```diff
@@ -1,15 +1,141 @@
-The docs are built on GitHub Actions and run the ``.ci_scripts/update_docs`` script.
+## Getting Started
 
-To build the docs locally, follow the steps mentioned below:
- 1.  Clone the repository into your local machine.
- 2.  Go into the main folder and run the following commands.  
- 3. ``conda env create -f ./.ci_scripts/environment.yml``
- 4. ``conda activate conda-forge-docs``
- 5. ``cd newsfeed && pip install --no-deps .``
- 6. ``cd ../src``
- 7. ``make html``
+The docs are built on GitHub Actions and run the `.ci_scripts/update_docs` script.
 
-Documentation update PRs should only change the files in ``src``. 
+### Prerequisites
 
-Once the PR is merged to ``master``, you can find the auto-generated html files in ``docs`` directory.
+#### Install
 
+You will need to have [Anaconda](https://problemsolvingwithpython.com/01-Orientation/01.03-Installing-Anaconda-on-Windows/) installed on your machine to use conda as a package manager
```

Comment:
You could link to the Anaconda install docs instead.

https://docs.anaconda.com/anaconda/install/windows/

Reply:
Alright thank you, on it

Reply:
Please use miniforge instead of anaconda.

Reply:
Alright, please can I get the link for installing the miniforge ? @isuruf   Thanks.

Reply:
@isuruf I found this [link](https://github.com/conda-forge/miniforge/releases) which contains a miniforge installers
and also this [link](https://github.com/conda-forge/miniforge).

Adding the above can make the steps more complicated for a beginner, but if you can point me to the right source I can work it out.

Reply:
I would go with the second link you listed, since it has install instructions below.